### ISSUE TITLE
Canon EOS M50 LCD screen goes black

### ISSUE THREAD

**Issue by Rwide** [gphoto/gphoto2#472]
Hello and thank you for this great tool! =)

I am using EOS M50 and the command "gphoto 2 --filename '/home/pi/photos/%Y-%m-%d %H-%M-%S %f.%C' --capture-image-and-download" on a raspberry pi 4 with node-red web interface.. This worked great for a while, resulting in an image review on the camera lcdscreen and a jpg picture on the Raspberry pi, great! Then all of a sudden the camera freezed and after that it doesnt work anymore.

What happends now is that the camera still takes pictures and delivers them to the raspberry pi when using the same command, but the camera lcdscreen goes black. No liveview, no image review. Just black =(

I have tried sending the camera to service. Canon sent it back saying there is nothing wrong with it. 
I have tried canon M50 markII-same result. I have also tried Sony A7III, Panasonic/Lumix S5, Canon EOS RP, Nikon D3500.  I could not get any of them to deliver image review or live view on camera lcdscreen while using the command above. 
Now I am kind of stuck. Any suggestions?

**Name the camera**
Canon EOS M50

Not sure what version of gphoto2 I have but I can check later. 
The camera is currently on service so I can't produce a logfile atm, but maybe later. Not sure how to do it though.

![IMG_20210515_133241](https://user-images.githubusercontent.com/95875820/145450005-b2b617b8-6f7b-44b3-94c9-184abc42a04c.jpg)
![Snap 2021-12-09 at 18 55 10](https://user-images.githubusercontent.com/95875820/145450127-ee855d88-ea0e-4efd-b91b-7a0f03d873c8.png)

**Comment by msmeissn** [comment]
did you update the software at that time?

can you try current git master branch again? (or 2.5.28 once released)

**Comment by Rwide** [comment]
No, I did not update the software at that time. I just updated to the latest version and now it's worse than before =(. It will only take one photo and then I have to unplug the usb and put it back in before I can take another photo. Screen still goes black every time it triggers..

**Comment by Rwide** [comment]
![gphoto2 Canon m50](https://user-images.githubusercontent.com/95875820/151646546-56d38c80-bc3c-4b58-975a-cc72d40feb25.jpg)

Flashed the raspberry and reinstalled fresh raspian os and gphoto2. Now the camera takes pictures again =). Still no liveview or image review on camera screen. But it's not black anymore. Now it shows an image of a computer on the screen. I would rather have liveview and/or image review od course =(.

**Comment by SystemDiagnosticss** [comment]
@Rwide Are you solved?
